code/modules/mob/living/simple_animal/borer/egg.dm

```diff
@@ -71,7 +74,9 @@
 		qdel(src)
 	else
 		src.visible_message("<span class='notice'>\The [name] calms down.</span>")
-		Grow() // Reset egg, check for hatchability.
+		hatching = FALSE
+		spawn (BORER_EGG_RERECRUITE_DELAY)
```

Comment:
No it isn't you idiot because I added this line myself.

We haven't come to an agreement yet so new code is free to be whatever, changing existing lines is wrong.